workflow/rules/ref.smk

```diff
@@ -13,7 +12,59 @@ rule get_transcriptome:
         type="cdna|cds|ncrna",
     cache: "omit-software"
     wrapper:
-        "0.74.0/bio/reference/ensembl-sequence"
+        "v1.7.1/bio/reference/ensembl-sequence"
+
+
+if is_3prime_experiment:
```

Comment:
Move this into a separate file ref_3prime.smk

Reply:
Moved.